csr_check_test.go

```diff
@@ -1220,8 +1402,7 @@ func Test_authorizeCSR(t *testing.T) {
 						},
 					},
 				},
-				nodeName: "bear",
-				nodeErr:  errors.NewNotFound(schema.GroupResource{}, ""),
+				node: withName("bear", defaultNode()),
```

Comment:
i'm curious why some of these entries have the `nodeErr` removed?

Reply:
These tests need a lot of refactoring. These test cases as you can see contain no changes aside from this, and they don't fail. So it was initially a copy-paste job, and it is a dead code.